docs/standards/smirnoff.md

```diff
@@ -331,6 +331,10 @@ Different cut-off treatments can be applied to periodic and non-periodic systems
 * `"cutoff"`: The vdW interaction is truncated at a distance specified by the `cutoff` attribute.
 * `"no-cutoff"`: The vdW interaction is not truncated.
 
+`periodic_method` can take additionally take the following values:
+
+* `Ewald3D`: a method like [particle mesh Ewald](http://docs.openmm.org/development/userguide/theory/02_standard_forces.html#coulomb-interaction-with-ewald-summation) should be used. This is only compatible with `potential="Lennard-Jones-12-6"`.
```

Comment:
Shouldn't we just say "Ewald" instead of "particle mesh Ewald" here, and link to a stable description, like `http://docs.openmm.org/8.0.0/userguide/theory/02_standard_forces.html#coulomb-interaction-with-ewald-summation`? We can mention that PME can be used as an implementation detail.

I thought we decided this is compatible with every potential that employs an 1/r^6, but we may restrict it to LJ-12-6 in the current implementations of the spec?

Reply:
Yeah, I agree with not restricting this to LJ-12-6. There are a lot of nuances to what we could put here but I'd have a slight preference for removing the last sentence altogether, implicitly reserving the right to raise NotImplementedErrors on the software level.

Reply:
> Shouldn't we just say ...

Yup - I'm going with "Ewald summation" to be slightly more explicit

> I thought we decided this is compatible with every potential that employs an 1/r^6

That's my understanding as well

> we may restrict it to LJ-12-6 in the current implementations of the spec?

That is precisely what I intend here - future versions of the spec that include LJ variants or other functional forms altogether should discuss their compatibility with 12-6 Lennard-Jones. In all likelihood it is as simple as the restrictions you describe, but I feel strongly that this proposal should elaborate on compatibility issues with unsupported potentials. In the "Usage an Impact" section I attempt to encode this:

> In this first iteration, `periodic_method="Ewald3D"` is only compatible with `potential="Lennard-Jones-12-6"`, which is currently the only supported value. Future changes to the `potential` attribute should discuss compatibility with LJPME, if any, including which terms can be ignored in reciprocal space.